components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -51,6 +52,7 @@ void RRTMGPRadiation::set_grids(const std::shared_ptr<const GridsManager> grids_
   add_field<Required>("eff_radius_qi", scalar3d_layout_mid, micron, grid->name());
 
   // Set computed (output) fields
+  add_field<Computed>("T_mid"     , scalar3d_layout_mid, K  , grid->name());
```

Comment:
There is now a new `add_field` type, `Updated` which stands for fields that are both required and computed.  Since `T_mid` is now both you could remove the `Required` declaration above and switch this to `Updated`.

Reply:
Fixed, thanks for the tip!